packages/core-services/tests/02.wallet/db.ts

```diff
@@ -0,0 +1,155 @@
+import { IDatabase, IWallet } from '@cypherock/db-interfaces';
+import { mockDb } from '../__mocks__/mocks';
+import { insertWallets, updateWallets, deleteWallets } from '../../src';
+
+describe('Wallet Operations', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('insertWallets', () => {
+    test('should insert wallets into the database', async () => {
+      const wallets: IWallet[] = [
+        {
+          deviceId: '1',
+          name: 'Wallet 1',
+          hasPassphrase: false,
+          hasPin: false,
+        },
+        {
+          deviceId: '2',
+          name: 'Wallet 2',
+          hasPassphrase: false,
+          hasPin: false,
+        },
+      ];
+
```

Comment:
All the test cases should be in `fixtures`. Also, the fixture should allow for multiple such cases.